src/object_properties.h

```diff
@@ -61,6 +61,7 @@ struct ObjectProperties
 	float eye_height = 1.625f;
 	float zoom_fov = 0.0f;
 	bool use_texture_alpha = false;
+	v3f light_anchor = v3f(0.0f, 0.6f, 0.0f);
```

Comment:
This value should only be the default for players. For some objects this could be above their actual selection-/collision box